common/lib/src/realm_types.dart

```diff
@@ -19,6 +19,7 @@
 import 'dart:ffi';
 import 'dart:typed_data';
 import 'package:objectid/objectid.dart';
+import 'package:uuid/uuid.dart';
```

Comment:
I'm not too keen on this package. Try looking into the ctor `UuitValue.fromList` it basically converts the bytes to a `String` and the parses it back. I think that is unreasonably slow for our use.

Reply:
We talked about this. There is no denying that the [uuid](https://pub.dev/packages/uuid) package is the more popular package. However its internal state is a string, which is pretty bad for us (really anybody), since we need to convert going back and forth.

However, we have options. One being simply implementing the interface of `UuidValue` in an efficient manner, at some point down the line.

Reply:
Discussed offline - we agree that the package as it is is not great, but it's likely to be widely used and the functionality is there, even if the actual implementation is not super performant. We agreed to go with the existing package and try to submit a PR to the maintainer that optimizes the UuidValue construction.